### ISSUE TITLE
Dashboard does not meet wcag 2.1 AA standards

### ISSUE THREAD

**Issue by tillydray** [interval/interval-node#26]
I ran Firefox's accessibility tool as well as [Deque's axe DevTools](https://github.com/dequelabs/axe-core) and they each reported a number of issues. For reference, wcag 2.1 guidelines can be seen [here](https://www.w3.org/TR/WCAG21/)

**Comment by jacobmischka** [comment]
Thank you for bringing this to our attention! We'll investigate and revise some of our color choices to ensure readability as soon as possible.
